airflow/providers/google/cloud/triggers/dataplex.py

```diff
@@ -0,0 +1,104 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""This module contains Google Dataplex triggers."""
+from __future__ import annotations
+
+import asyncio
+from typing import AsyncIterator, Sequence
+
+from google.cloud.dataplex_v1.types import DataScanJob
+
+from airflow.providers.google.cloud.hooks.dataplex import DataplexAsyncHook
+from airflow.triggers.base import BaseTrigger, TriggerEvent
+
+
+class DataplexJobTrigger(BaseTrigger):
```

Comment:
This name and the docs don't seem to indicate that this is actually for Data Quality scans.

Reply:
currently this trigger does not exist